recipe/meta.yaml

```diff
@@ -28,6 +28,11 @@ outputs:
         - python
       run:
         - python
+        # While libpython is an implicit dependency and will thus be installed
+        # regardless of whether or not it's included here, it's also a direct
+        # dependency of aesara, so we mention it explicitly.
+        - libpython >=2.0  # [win]
```

Comment:
It looks like [`libpython`](https://anaconda.org/conda-forge/libpython) is particular to `mingw-w64` on Windows.  With that in mind, would it make sense to include `m2w64-toolchain`, in case having one without the other doesn't make sense.  

Also, how bold is the `mingw-w64` choice in general?  We've had problems with Windows installations due to a lack of such a choice, so this is worth addressing.

(I'm assuming that none of this applies to Linux/Mac.)

Reply:
Yes, as the base package should not force anything on the compile environment, `libpython` should not be a dependency there.